pkg/didcomm/dispatcher/outbound.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package dispatcher
+
+import (
+	"errors"
+
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/transport"
+)
+
+// OutboundDispatcher dispatch msgs to destination
+type OutboundDispatcher struct {
+	outboundTransport transport.OutboundTransport
```

Comment:
outbound transport cannot be set before inspecting the endpointURL contained in the `*Destination` passed to the `Send()` method

Reply:
Done I add array of outboundTransport and add type method